tests/test_docs.py

```diff
@@ -162,6 +162,10 @@ def run_block_test(self, block):
                 pass
             elif block.lang == 'graphql':
                 graphql_parser.parse(block.code)
+            elif block.lang == 'graphql-schema':
+                # The graphql-schema can be highlighter using graphql
```

Comment:
can be highlighter -> can be highlighted

Reply:
Fixed